Issues when installing on Windows 11

I'm running `install_env.ps1` from a Anaconda PowerShell Prompt on Windows 11.

First issue encountered is a failure to build `fused_ssim`. This is the error:
```
Building wheels for collected packages: fused_ssim

...

      The detected CUDA version (12.1) mismatches the version that was used to compile
      PyTorch (11.8). Please make sure to use the same CUDA versions.

      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
  ERROR: Failed building wheel for fused_ssim
  Running setup.py clean for fused_ssim
Failed to build fused_ssim
ERROR: Failed to build installable wheels for some pyproject.toml based projects (fused_ssim)
Error: Requirements installation failed with exit code 1
```
I managed to resolve it by running `conda install -c "nvidia/label/cuda-11.8.0" cuda-toolkit` as mentioned [here](https://github.com/bitsandbytes-foundation/bitsandbytes/issues/212). Unfortunately a new problem appeared: `fatal error C1189: #error:  -- unsupported Microsoft Visual Studio version! Only the versions between 2017 and 2022 (inclusive) are supported! The nvcc flag '-allow-unsupported-compiler' can be used to override this version check; however, using an unsupported host compiler may cause compilation failure or incorrect run time execution. Use at your own risk.`

I managed to resolve it by running `conda install -c anaconda vs2019_win-64` as discussed [here](https://github.com/graphdeco-inria/gaussian-splatting/issues/833#issuecomment-2627463581) and [here](https://github.com/graphdeco-inria/gaussian-splatting/issues/146#issuecomment-1718015437).

Then manually continuing the steps in `install_env.ps1`.
```
pip install -r requirements.txt
pip install cython
pip install hydra-core
pip install https://nvidia-kaolin.s3.us-east-2.amazonaws.com/torch-2.5.1_cu118/kaolin-0.17.0-cp311-cp311-win_amd64.whl
pip install -e .
```

At this point I was able to train, but still not able to run the playground. Due to the following error which was also discussed [here](https://github.com/nv-tlabs/3dgrut/issues/103):
```
Creating library libplayground_cc.lib and object libplayground_cc.exp
hybridTracer.o : error LNK2019: unresolved external symbol "void __cdecl computeMeshFaceBuffer(unsigned int,struct float3 const * __restrict,struct int3 const * __restrict,struct float3 *,struct int3 *,struct CUstream_st *)" (?computeMeshFaceBuffer@@YAXIPEIBUfloat3@@PEIBUint3@@PEAU1@PEAU2@PEAUCUstream_st@@@Z) referenced in function "public: void __cdecl HybridOptixTracer::buildMeshBVH(class at::Tensor,class at::Tensor,unsigned int,bool)" (?buildMeshBVH@HybridOptixTracer@@QEAAXVTensor@at@@0I_N@Z)
  Hint on symbols that are defined and could potentially match:
    "void __cdecl computeMeshFaceBuffer(unsigned int,struct float3 const *,struct int3 const *,struct float3 *,struct int3 *,struct CUstream_st *)" (?computeMeshFaceBuffer@@YAXIPEBUfloat3@@PEBUint3@@PEAU1@PEAU2@PEAUCUstream_st@@@Z)
libplayground_cc.pyd : fatal error LNK1120: 1 unresolved externals
ninja: build stopped: subcommand failed.
```

I could resolve it by adding `__restrict__` to the `verts` and `faces` arguments of `computeMeshFaceBuffer` in `threedgrut_playground/src/meshBuffers.cu`. Now the playground also works.

In an earlier attempt I encountered the issue of `cuda.lib` not being found, which I managed to resolve by adding `"/LIBPATH:C:\\Users\\username\\AppData\\Local\\anaconda3\\envs\\3dgrut\\Library\\lib"` to the `ldflags` in `threedgut/utils/jit.py` and `threedgrt_tracer/setup_3dgrt.py`, however this was not necessary in my second attempt.

Hopefully the above can help people resolve some issues and to improve the installation process on Windows.

Thanks, I tried to build it again using #125 . 

With the PR, it's no longer required to patch `threedgrut_playground/src/meshBuffers.cu` before running the playground, so that's a nice improvement.

I ran into `fatal error C1189: #error:  -- unsupported Microsoft Visual Studio version! Only the versions between 2017 and 2022 (inclusive) are supported! The nvcc flag '-allow-unsupported-compiler' can be used to override this version check; however, using an unsupported host compiler may cause compilation failure or incorrect run time execution. Use at your own risk.` again, which I was able to 'solve' this time by removing VS2022. I don't think it's related to your PR, just Windows environment shenanigans. After removing VS2022, I still need to run `conda install -c "nvidia/label/cuda-11.8.0" cuda-toolkit`, but `conda install -c anaconda vs2019_win-64` was not required this time.

--- a/install_env.ps1
+++ b/install_env.ps1
@@ -33,7 +33,11 @@ function Find-VisualStudioCompiler {
         "C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\*\bin\Hostx64\x86",
         "C:\Program Files\Microsoft Visual Studio\2022\Professional\VC\Tools\MSVC\*\bin\Hostx64\x86", 
         "C:\Program Files\Microsoft Visual Studio\2022\BuildTools\VC\Tools\MSVC\*\bin\Hostx64\x86",
-        "C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\*\bin\Hostx64\x86"
+        "C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\*\bin\Hostx64\x86",
+        "C:\Program Files (x86)\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\*\bin\Hostx64\x86",
+        "C:\Program Files (x86)\Microsoft Visual Studio\2022\Professional\VC\Tools\MSVC\*\bin\Hostx64\x86", 
+        "C:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\VC\Tools\MSVC\*\bin\Hostx64\x86",
+        "C:\Program Files (x86)\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\*\bin\Hostx64\x86"
     )
     
     foreach ($path in $searchPaths) {

